src/components/molecules/CardItem/style.tsx

```diff
@@ -69,33 +71,38 @@ export const Wrapper = styled.article`
       font-family: 'Noto Sans KR Bold';
     }
   }
-
-  button {
-    margin-top: 24px;
-    border-radius: 4px;
-    padding: 6px 12px;
-    margin-inline-end: 8px;
-
-    font-family: 'Noto Sans KR Medium';
-    font-size: 14px;
-    line-height: 20px;
-    text-align: center;
-    cursor: pointer;
-
-    &.request {
-      width: 108px;
-      height: 32px;
-      border: none;
-      background-color: #2196f3;
-      color: #ffffff;
-    }
-
-    &.contact {
-      width: 80px;
-      height: 32px;
-      border: 1px solid #2196f3;
-      background-color: #ffffff;
-      color: #2196f3;
+  div {
+    display: flex;
+    button {
+      margin-top: 24px;
+      border-radius: 4px;
+      padding: 6px 12px;
+      margin-inline-end: 8px;
```

Comment:
이 부분 안 그래도 명세서 상의 `padding-top`이 너무 과한 것 같아서 4px로 줄이려고 했어요...
혹시 아래 제안 드린 버전으로 고쳐주실 수 있을까요? 🙏

```suggestion
      margin-top: 24px;
      margin-inline-end: 8px;
      border-radius: 4px;
      padding: 4px 12px;
```

Reply:
padding-top 은 수정사항이 없고, 단지 상위 div에 display: flex를 추가하면서 buttons를 div 하위로 옮겨준게 다입니다~!